javascript/manage_custom_field_edit_page.js

```diff
@@ -0,0 +1,19 @@
+// Handle switching between textarea custom field type and other types
```

Comment:
Add the standard header (see common.js)

Not sure what the strategy is wrt javascript files - in some cases we have dedicated files (e.g. login.js, bugFilter.js) and in others the code just goes into common.js. Maybe we should define that as part of our guidelines ?

In any case, would it not make sense to name the file in a somewhat more generic manner (e.g. custom_field.js) in case we later need to reuse it elsewhere ?
Should the file name be more specific